cartography/intel/spacelift/ec2_ownership.py

```diff
@@ -235,18 +257,25 @@ def sync_ec2_ownership(
     update_tag: int,
     account_id: str,
 ) -> None:
-
+    """
+    Sync EC2 ownership data from CloudTrail into Neo4j as CloudTrailEvent nodes.
+    """
     logger.info("Starting EC2 ownership sync")
 
     cloudtrail_data = get_ec2_ownership(aws_session, bucket_name, object_prefix)
 
-    mappings = transform_ec2_ownership(cloudtrail_data)
+    events = transform_ec2_ownership(cloudtrail_data)
 
-    if mappings:
-        load_ec2_ownership_relationships(
-            neo4j_session, mappings, update_tag, account_id
-        )
+    if events:
+        load_cloudtrail_events(neo4j_session, events, update_tag, account_id)
     else:
-        logger.warning("No EC2 ownership mappings found - no relationships created")
+        logger.warning("No CloudTrail events found - no nodes created")
+
+    common_job_parameters = {
+        "UPDATE_TAG": update_tag,
+        "SPACELIFT_ACCOUNT_ID": account_id,
+        "account_id": account_id,
```

Comment:
If in future we support spacelift things with other providers, the name 'account_id' could be confusing so maybe we should rename this to 'aws_account_id'